app/uk/gov/hmrc/gform/views/form/snippets/lookup_autosuggest.scala.html

```diff
@@ -60,7 +60,7 @@
       <div
       class='lookup @{if(hasErrors) "gform-lookup-error" else ""}'
       data-field-id="@fcId"
-      data-basefield-id="@{formComponentId.baseComponentId.value}"
+      data-basefield-id="@{formComponentId.value}"
```

Comment:
@josef-vlach @DursunGundogan 
I am not sure why baseComponentId was used in the first place. If there is a valid reason for that I think we might need to change lookup API to take both basecomponentId and current componentId.

Reply:
I'm not sure why based is used, but if you are changing this, make sure you are propagating the change till the end.

Reply:
Having said that, I think it uses "base" as it was never required to support ATL

Reply:
do you mean to rename  baseComponentId to componentId to avoid confusion? The autocompletion works as it is now.

Reply:
I can see that `data-basefield-id` is used in `hmrc-gform-prototype` project. 
I submitted a PR to this project . Interestingly this is a first PR there
https://github.com/hmrc/hmrc-gforms-prototype/pull/1
Looks like Tom just push the changes to main